src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbol.cs

```diff
@@ -223,15 +226,14 @@ public override SyntaxList<AttributeListSyntax> AttributeDeclarationSyntaxList
 
                     if (accessor.Body != null || accessor.ExpressionBody != null)
                     {
-                        isAutoProperty = false;
                         accessorsHaveImplementation = true;
                     }
                 }
             }
             else
             {
-                isAutoProperty = false;
                 accessorsHaveImplementation = GetArrowExpression(syntax) is object;
+                Debug.Assert(accessorsHaveImplementation); // it's not clear how this even parsed as a property if it has no accessor list and no arrow expression.
```

Comment:
I didn't understand the comment. `int P { }` is parsed as a property
[https://sharplab.io/#v2:EYLgtghglgdgNAFxBAzggPgWAFAAEDMABLgEyEDChA3joXcUbAoQArWEC+OHQA==](https://sharplab.io/#v2:EYLgtghglgdgNAFxBAzggPgWAFAAEDMABLgEyEDChA3joXcUbAoQArWEC+OHQA==)

It would be good to include a test for that in partial properties

Reply:
The property in your example has an empty accessor list.

Reply:
Got it. Thank you